JavaScript/conditional-operator.md

```diff
@@ -2,28 +2,31 @@
 
 ###What is it?
 
-The ternary operator(also called the conditional operator) allows us to return different values depending on the results of checks.
+The ternary operator (also called the conditional operator) allows us to return different values depending on the results of checks.
 It can be a cleaner and shorter alternative to the `if...else` statement.
 
 ###General syntax
 
 ```javascript
-var test= condition ? expr1 : expr2 
+var test = condition ? expr1 : expr2;
 ```
 
 `if...else` equivalent:
 
 ```javascript
-if (condition===true) {test=expr1} 
-else {test=expr2}
+if (condition === true) {
```

Comment:
I don't think the "=== true" is required since it isn't included in the conditional operator example